ceph-releases/luminous/ubuntu/16.04/daemon/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y wget unz
     wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add - && \
     echo "deb http://download.ceph.com/debian-$CEPH_VERSION/ xenial main" | tee /etc/apt/sources.list.d/ceph-$CEPH_VERSION.list && \
     echo "deb http://download.ceph.com/nfs-ganesha/deb-V2.5-stable/luminous/ xenial main" | tee /etc/apt/sources.list.d/nfs-ganesha.list && \
-    apt-get update && apt-get install -y --force-yes $PACKAGES && \
+    apt-get update && apt-get install -y  --no-install-recommends --force-yes $PACKAGES && \
     dpkg -s $PACKAGES && \
     apt-get clean && rm -rf $PURGES && \
```

Comment:
can you include this part in your clean up script?

Reply:
I don't think that's a good idea since that would make that layer bigger than expected

Reply:
then let's put everything in one `RUN` statement :)

Reply:
yes !